src/aiida/cmdline/params/options/main.py

```diff
@@ -592,6 +593,13 @@ def set_log_level(_ctx, _param, value):
     help='Only include entries created before OLDER_THAN days ago.',
 )
 
+MOST_RECENT_NODE = OverridableOption(
+    '-M',
+    '--most-recent-node',
+    is_flag=True,
```

Comment:
`default = False` here not required?

Reply:
For boolean flags without an off-switch, this is not necessary: https://click.palletsprojects.com/en/8.1.x/options/#boolean-flags

Reply:
Yeah, makes a lot of sense. We just got a bit confused as for example in the `--all` option below the default is given, but I guess this is redundant?